duckduckgo/BrowserServicesKit
# Remove content scope files causing warnings

Task/Issue URL: https://app.asana.com/0/1200194497630846/1201640066177016/f
Tech Design URL:
CC:

**Description**:

- This removes the warnings when building.
- I couldn't find a way to make this work without requiring all the hard coded files (without turning the content scope scripts into a package which we probably will want long term https://app.asana.com/0/1201614831475344/1201638298093318/f)

**Steps to test this PR**:
1. Open up XCode and build
2. Check for no warnings



**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
